cli/command/registry_test.go

```diff
@@ -1,13 +1,17 @@
 package command_test
 
 import (
+	"bytes"
 	"testing"
 
 	"github.com/pkg/errors"
 	"github.com/stretchr/testify/assert"
 	"golang.org/x/net/context"
 
 	// Prevents a circular import with "github.com/docker/cli/internal/test"
+
+	"fmt"
```

Comment:
nit: this import should be on top :angel:

Reply:
Fixed